apps/web/src/components/HintsAccordion/HintsAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Heading,
+} from "@chakra-ui/react";
+
+import { useColor } from "../../styles/useColor";
+import { type SignPage } from "../Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+
+export const Hints: Record<SignPage, HintData> = {
+  TezSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
```

Comment:
```suggestion
      "A confirmation means that the block is confirmed by the network but the operation can still be cancelled due to chain reorganizations. Finality means that the operation is irreversible.",
```
Attack is incorrect here. With Tenderbake, you can have a alternative heads scenario, where a block payload proposal generated on (levl n, round k+1) is not confirmed and the chosen chain is a block on (level n+1, usually round 0) which is  a succesor of (level n,  round k (or earlier)). 

This  might just happen out of slow consensus/ degraded health / network congestion scenarios -- which they don't necessarily entail an attack.